doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -519,17 +521,17 @@ See the following instructions.
 
       Use the ``--recover`` parameter with ``west flash``, as described in :ref:`programming_params`.
 
-   .. group-tab:: nrfjprog
+   .. group-tab:: nRF Util
 
       Enter the following commands to recover first the network core and then the application core::
 
-        nrfjprog --recover --coprocessor CP_NETWORK
-        nrfjprog --recover
+        nrfutil device recover --core Network
+        nrfutil device recover
```

Comment:
I dont know what goes wrong here, but if I try to combine the nrfutil recover commands with the VERIFY_READ command, I get errors. 
![image](https://github.com/nrfconnect/sdk-nrf/assets/89006246/07b51822-cfec-48de-85ba-f09e97f8e396)

Reply:
For comparison: `west flash` works fine just after

Reply:
@bjorgan said that the fix to https://github.com/nrfconnect/sdk-nrf/pull/14348/files#r1511464831 should also fix this issue.